TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>Should also add some commentary here about what we can find in a fits file, then a quick discussion after the fact about what we see. Even if we've covered this stuff before, it's the "bread and butter" of writing these types of Notebooks</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>added both</p><p> </p>